com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Settings/FrameSettings.cs

```diff
@@ -334,9 +334,9 @@ public enum FrameSettingsField
         /// <summary>When enabled, HDRP uses FPTL for forward opaque.</summary>
         [FrameSettingsField(3, autoName: FPTLForForwardOpaque, tooltip: "When enabled, HDRP uses FPTL for forward opaque.")]
         FPTLForForwardOpaque = 120,
-        /// <summary>When enabled, HDRP uses a big tile prepass for light visibility.</summary>
-        [FrameSettingsField(3, autoName: BigTilePrepass, tooltip: "When enabled, HDRP uses a big tile prepass for light visibility.")]
-        BigTilePrepass = 121,
+        /// <summary>When enabled, HDRP performs binning to determine the set of visible lights.</summary>
+        [FrameSettingsField(3, autoName: BinnedLighting)]
```

Comment:
isn't this only for the case of deffered? shouldn't we call this option deferred Binned lighting?